marklogic-data-hub/src/main/java/com/marklogic/hub/impl/EntityManagerImpl.java

```diff
@@ -150,12 +150,11 @@ public boolean saveDbIndexes() {
             List<JsonNode> entities = getAllEntities();
 
             if (entities.size() > 0) {
-                DbIndexGenerator generator = new DbIndexGenerator(hubConfig.newReverseFlowClient());
-                String indexes = generator.getIndexes(entities);
+                DbConfigsManager generator = new DbConfigsManager(hubConfig.newReverseFlowClient());
+                ObjectNode indexNode = generator.generateIndexes(entities);
```

Comment:
i think that we should call release() in a finally block for the database client object.